keyboards/comet46/rules.mk

```diff
@@ -64,3 +29,7 @@ UNICODE_ENABLE = YES 		# Unicode
 # BLUETOOTH_ENABLE = yes # Enable Bluetooth with the Adafruit EZ-Key HID
 
 USB = /dev/ttyACM0
+# project specific files
+SRC += matrix.c \
+       i2c.c \
```

Comment:
i2c.c should be added via `QUANTUM_SRC_LIB`, to avoid issues.  

Howver, that may be outside the scope of this PR.